tests/ui/rfcs/rfc-3348-c-string-literals/issue-113235.rs

```diff
@@ -0,0 +1,7 @@
+// Regression test for issue #113235.
+// check-pass
+// edition:2015
```

Comment:
Fyi, I know that the default edition of UI tests is 2015 but I feel like making it explicit is more robust and telling.
I can spruce up this regression test with an additional comment if you'd like me to.